cg/store/api/findbusinessdata.py

```diff
@@ -317,6 +318,20 @@ def pool(self, pool_id: int) -> models.Pool:
         """Fetch a pool."""
         return self.Pool.get(pool_id)
 
+    def ready_made_library_expected_reads(self, case_id: str) -> float:
+        """Return the target reads of a ready made library case"""

```

Comment:
```suggestion
        """Return the target reads of a ready made library case."""
```
Apparently docstrings should end with a dot according to python standards. Who could have known....